storefront/app/helpers/workarea/storefront/application_helper.rb

```diff
@@ -90,6 +90,39 @@ def optional_field(prompt, *fields, &block)
         content_tag(:div, capture(&block), { class: 'hidden-if-js-enabled',
                                              data: { optional_field: prompt } })
       end
+
+      def lazy_image_tag(source, options = {})
+        options = options.symbolize_keys
+
+        check_for_image_tag_errors(options)
+        skip_pipeline = options.delete(:skip_pipeline)
+
+        if options[:srcset] && !options[:srcset].is_a?(String)
+          options[:srcset] = options[:srcset].map do |src_path, size|
+            src_path = path_to_image(src_path, skip_pipeline: skip_pipeline)
+            "#{src_path} #{size}"
+          end.join(", ")
+        end
+
+        options[:width], options[:height] = extract_dimensions(options.delete(:size)) if options[:size]
+
+        image_source = resolve_image_source(source, skip_pipeline)
+        if options.fetch(:lazy_load, true)
```

Comment:
Wouldn't this always be true since you're calling `lazy_image_tag`? If you didn't want, you'd just call `image_tag`.

Reply:
Brraaaiiiinnnnssss